components/locale.js

```diff
@@ -61,28 +61,47 @@ function getInitialLocaleProps({req} = {}) {
   return Promise.resolve({locale})
 }
 
-// eslint-disable-next-line complexity
 function getContent(locale, options) {
-  const {page, component, example} = options
+  const fallbackContent = content('', options)
+  if (locale === fallbackLocale) {
+    return fallbackContent
+  }
+  const localeContent = content(`${locale}/`, options)
+  return Object.keys(fallbackContent).reduce((cont, key) => {
+    const value = localeContent[key]
+    cont[key] = value === null ? null : value || fallbackContent[key]
+    return cont
+  }, {})
+}
+
+// eslint-disable-next-line complexity
+function content(localePath, options) {
+  const {
+    page,
+    pagePath = (path, p) => `../pages/${p}/content/${path}index.js`,
+    component,
+    componentPath = (path, c) => `../components/content/${path}${c}`,
+    example,
+    examplePath = (path, e) => `../examples/content/${path}${e}`,
+  } = options
```

Comment:
Nice. What if we did this instead:

```
const {
  page,
  component,
  example,
  contentPath = getContentPath()
} = options

function getContentPath() {
  if (options.page) {
    return 'blah'
  } else if (options.component) {
    return 'blah'
  } etc...
}
```

That way in the tests you could just specify `contentPath`. Thoughts?